API-8379 add info about new reports (#911)
livechat/livechat-public-docs: content/data-reporting/reports-api/v3.4/index.mdx

| **Required scopes** | -                                                            |

#### Request

See [available parameters and filters](#available-parameters-and-filters).

#### Response

| Field                    | Notes                                                    |
| ------------------------ | -------------------------------------------------------- |
| `total`                  | The total number of chats in the specified date range.   |
| `records`                | Contains the `distribution` objects, for example, `day`. |
| `records.day.chats`      | The total number of chats that `day`.                    |
| `records.day.bad`        | The number of chats rated `bad` that `day`.              |
| `records.day.good`       | The number of chats rated `good` that `day`.             |

</Text>
<Code>
<CodeSample path={'REQUEST'}>
<Sample>

```shell
curl -X POST \
https://api.livechatinc.com/v3.4/reports/chats/ratings \
  -H 'Content-Type: application/json' \
  -H 'Authorization: Bearer <your_access_token>' \
  -d '{
    "distribution": "day",
    "filters": {
      "from": "2021-06-23T00:00:00-00:00",

>       "to": "2021-06-27T23:59:59-00:00"

Reviewer: can we use the same date range as in other reports?

Author: changed, along with response

#### Specifics

|                     |                                                              |
| ------------------- | ------------------------------------------------------------ |
| **Method URL**      | `https://api.livechatinc.com/v3.4/reports/chats/ranking`     |
| **HTTP method**     | POST, GET                                                    |
| **Required scopes** | -                                                            |

#### Request

See [available parameters and filters](#available-parameters-and-filters).

#### Response

| Field                    | Notes                                                                                                                  |
| ------------------------ | ---------------------------------------------------------------------------------------------------------------------- |
| `ranking`                | Contains the `agent` objects.                                                                                          |
| `ranking.agent.total`    | The total number of chats an agent had in the specified period.                                                        |

> | `ranking.agent.good`     | The number of an agent's chats rated `good`                                                                            |

Reviewer 2: ```suggestion
| `ranking.agent.good`     | The number of an agent's chats rated `good`.                                                                            |
```

> | `ranking.agent.bad`      | The number of an agent's chats rated `bad`                                                                             |

Reviewer 2: ```suggestion
| `ranking.agent.bad`      | The number of an agent's chats rated `bad`.                                                                             |
```

| `ranking.agent.score`    | The agent's score. [Read more about how we calculate agent rankings...](https://www.livechat.com/help/agent-rankings/) |

</Text>
<Code>
<CodeSample path={'REQUEST'}>
<Sample>

```shell
curl -X POST \

> https://api.livechatinc.com/v3.4/reports/chats/ratings \

Reviewer: you mean ranking 😄

Author: 🤦

</Code>
</Section>

<Section>
<Text>

### Ratings

> Shows how many chats were rated and how they have been rated during specified period.

Reviewer 2: ```suggestion
Shows the number of rated chats along with their ratings during a specified period of time.
```

#### Response

| Field                    | Notes                                                    |
| ------------------------ | -------------------------------------------------------- |
| `total`                  | The total number of chats in the specified date range.   |
| `records`                | Contains the `distribution` objects, for example, `day`. |
| `records.day.chats`      | The total number of chats that `day`.                    |
| `records.day.bad`        | The number of chats rated `bad` that `day`.              |
| `records.day.good`       | The number of chats rated `good` that `day`.             |

</Text>
<Code>
<CodeSample path={'REQUEST'}>
<Sample>

```shell
curl -X POST \
https://api.livechatinc.com/v3.4/reports/chats/ratings \
  -H 'Content-Type: application/json' \
  -H 'Authorization: Bearer <your_access_token>' \
  -d '{
    "distribution": "day",
    "filters": {
      "from": "2021-06-23T00:00:00-00:00",
      "to": "2021-06-27T23:59:59-00:00"
    }
  }'
```

</Sample>
</CodeSample>

<CodeResponse type="reports" version="v3.4" json="ratings"/>

</Code>
</Section>

<Section>
<Text>

### Ranking

> Shows the ratio of good to bad ratings for each operator.

Reviewer 2: ```suggestion
Shows the ratio of good to bad ratings for each agent within a license.
```

#### Specifics

|                     |                                                              |
| ------------------- | ------------------------------------------------------------ |
| **Method URL**      | `https://api.livechatinc.com/v3.4/reports/chats/ranking`     |
| **HTTP method**     | POST, GET                                                    |
| **Required scopes** | -                                                            |

#### Request

See [available parameters and filters](#available-parameters-and-filters).

#### Response

| Field                    | Notes                                                    |
| ------------------------ | -------------------------------------------------------- |
| `ranking`                | Contains the `agent` objects.                            |

> | `ranking.agent.name`     | Name of `agent`                                          |

Reviewer 2: ```suggestion
| `ranking.agent.name`     | The agent's name.                                          |
```

> | `ranking.agent.total`    | The total number of agent's chats that period.           |

Reviewer 2: ```suggestion
| `ranking.agent.total`    | The total number of chats an agent had in the specified period.           |
```

> | `ranking.agent.good`     | The number of agent's chats rated `good`                 |

Reviewer 2: ```suggestion
| `ranking.agent.good`     | The number of an agent's chats rated `good`                 |
```

| `ranking.agent.bad`      | The number of agent's chats rated `good`                 |

> | `ranking.agent.score`    | Score of agent                                           |

Reviewer 2: ```suggestion
| `ranking.agent.score`    | The agent's score. [Read more about how we calculate agent rankings...](https://www.livechat.com/help/agent-rankings/)                                    |
```
